scripts/correlateTBN.py

```diff
@@ -18,7 +18,7 @@
 import argparse
 
 from lsl import astro
-from lsl.reader.ldp import LWA1DataFile
+from lsl.reader.ldp import LWA1DataFile, LWASVDataFile
```

Comment:
There is a `LWADataFile` function in `lsl.reader.ldp` that should handle all data formats from both stations.

Reply:
Great I'll change that.